spec/rental_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'spec_helper'
+
+describe Rental do
+  before :each do
+    @book = Book.new('Afghanistan', 'Abdulali')
+    @teacher = Teacher.new('math', 30, 'Salim')
+    @rental = Rental.new('2023-08-02', @book, @teacher, [0, 0])
+  end
+
+  context 'Add a rental' do
+    it 'Check if the rental is an object of the Rental class' do
+      expect(@rental).to be_an_instance_of Rental
+    end
+
+    it 'Check if the rental.book is an object of the Book class' do
+      expect(@rental.book).to be_an_instance_of Book
+    end
```

Comment:
- [ ] Please write unit tests for all the `to_hash` method in the `Rental` class as you are required to test all methods in all classes 👍